src/gui/accountsettings.cpp

```diff
@@ -403,6 +426,16 @@ void AccountSettings::slotOpenCurrentFolder()
     }
 }
 
+void AccountSettings::slotOpenCurrentLocalSubFolder()
+{
+    QModelIndex selected = ui->_folderList->selectionModel()->currentIndex();
+    if( !selected.isValid() )
```

Comment:
Normally, that function should only be called in this case. But it's true we could play it save.